onboarding/README.md

```diff
@@ -7,6 +7,7 @@ campus-access
 infrastructure
 dropbox-google-drive
 computer-setup
+eduroam
```

Comment:
maybe move 'eduroam' under computer-setup? i'm not sure what's more intuitive @joshuacwnewton @mguaypaq any suggestions?

Reply:
I tried that, but the `eduroam` section is relatively long. Thus I left it as a separate .md file.

Reply:
no worries-- in fact, it could stay as a separate MD file and be called as a sub-section. But let's wait what others think

Reply:
> in fact, it could stay as a separate MD file and be called as a sub-section. But let's wait what others think

I like this suggestion a lot. It's similar to how we have drop-down subsections here, as well:

![image](https://user-images.githubusercontent.com/16181459/201367465-a520bf3e-0d70-497e-9dda-bee352858acc.png)

We could have:

```
Onboarding \/
    Getting Started
    Campus Access
    Infrastructure
    Google Drive
    Computer Setup \/
        Mail Apps
        Eduroam
        Shell Profile
        Other configuration tips
    Developer Setup
    Consultants and RA
    Postdoctoral fellows
```

In other words, keep "mail" and "eduroam" as individual pages, but make them sub-pages under Computer Setup.

Reply:
> In other words, keep "mail" and "eduroam" as individual pages, but turn the sections into sub-pages under Computer Setup, that way they show up in the sidebar and should hopefully be slightly easier to find.

This sounds good, thank you 👍🏻 I split the `computer-setup.md` into sub-pages in https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/78/commits/98816c6908ed6d9610f4ac57f065df64a4a57c27